.travis.yml

```diff
@@ -32,6 +32,10 @@ before_install:
         conda install --yes scipy matplotlib nose coverage setuptools;
         travis_retry pip install -q coveralls;
       fi;
+    # install AVbin
+    - wget https://github.com/downloads/AVbin/AVbin/install-avbin-linux-x86-64-v10 -O avbin.sh
```

Comment:
The error is `Unable to establish SSL connection`, which suggests that it's an SSL protocol failure. Either a bug in Ubuntu/wget or GitHub's certs. Can you use http?